devlab.yml

```diff
@@ -0,0 +1,25 @@
+# DOCKER-VERSION 1.9.1
+# DEVLAB-VERSION 1.5.3
+
+#
+# Stardust DevLab YAML
+#
+# http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/
+# https://github.com/TechnologyAdvice/stardust/
+#
+
+from: node:argon
```

Comment:
Not only for consistency-- our container is about a fifth of the size of the official one. They insist on building on top of a full Debian install.